playbooks/incident-handling.md

```diff
@@ -0,0 +1,458 @@
+
+     > /remind me to add @new-engineer to on-call rotations in 60 days
+
+Considerations when adding a new engineer:
+
+- Avoid pairings of two new engineers. Place the new engineer earlier in the rotation to avoid this.
+
+- If for some reason you need to add a new engineer anywhere other than (a) where there was a `no one` placehold or
+  (b) the end of a rotation, the upcoming schedule of shifts will be modified, as the order of the participants
+  matters. **You should notify the affected engineers to avoid scheduling conflicts.** E.g.:
+
+  > Hi @impacted-engineer! We've added a new engineer to the on-call schedule which has modified the dates of your
+  > on-call shift. Please take some time to review the schedule and trade shifts as needed.
+
+- Alternatively, to minimize shifting multiple engineers' schedules, it may be preferable to reach out to one
+  engineer directly and ask them to switch shifts. E.g.:
+
+  > Hi @engineer! We've added a @new-engineer to the schedule and would like to support them by not pairing them
+  > with another new engineer for their first on-call shift. Would you be able to switch shifts with them? The new
+  > dates of your on-call shift would be X to Y.
+
+- Check for an unbalanced number of rotation participants in rot-a vs rot-b. If staff locations result in one
+  rotation having significantly fewer engineers, you should prioritize a balanced rotation schedule over Eastern
+  Time Zone coverage.
+
+</details>
+
+<details>
+  <summary>Removing engineers from the schedule</summary>
+
+When an engineer leaves:
+
+- They will be replaced with a `no one` placeholder in the schedule as part of the
+  [off-boarding checklist](https://docs.google.com/document/d/10mmqkXnYVp0ZOmF5JwVPyxkzcyIN-rqO8prHgIi11lw/edit)🔒.
+- Their manager will be asked to override that shift with themselves.
+- Their Opsgenie user account will be deleted.
+  </details>
+
+<details>
+  <summary>Trading or overriding shifts</summary>
+
+To switch shifts you may schedule an override in OpsGenie. To override a shift:
+
+1. Navigate to
+   [Engineering On-Call Schedule](https://artsy.app.opsgenie.com/settings/schedule/detail/aa7d47fa-ab67-4c09-b9ca-d43a20efbb04)🔒
+
+1. Find and hover over the shift and click override:
+
+
+    <img width=500 src="images/opsgenie-override-0.png" />
+
+1. You may want to update the default configuration:
+
+- By default, you will take the shift. If you'd like to assign someone else, search for and assign them
+- By default, you will override the entire shift. If you'd like to partially override the shift, adjust the
+  Starts/End on dates
+
+   <img width=500 src="images/opsgenie-override-1.png" />
+
+1. Click "Add" and verify your override:
+
+      <img width=500 src="images/opsgenie-override-2.png" />
+   </details>
+
+<details>
+  <summary>Special holiday shifts</summary>
+
+Holiday on-call shifts will take place from December 21st to Janurary 6th, these holiday rotations are labeled
```

Comment:
#minor but since I imagine the actual dates here will change based on when Christmas falls during the week, we could make this more generic to avoid confusion.